t/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: "3.5"
```

Comment:
we also need to mount the dgraph binary inside the docker container.

Reply:
actually this is not necessary because the dgraph binary is embedded in the docker image we just built.  There is no reason to make this more complicated by injecting the binary @skrdgraph

Reply:
+1 ^

Reply:
But when I am testing things locally, I won't be able to choose the dgraph binary. This is not normal, and this would cause confusion.

Reply:
@mangalaman93 I think there is a misunderstanding.  To do what you want to do (build dgraph locally from source, build a dgraph image, run dgo tests against that image/binary) you can still do.  All you need to do is
```
make docker-image // build dgraph binary locally from source, and make docker image with local tag
DGRAPH_VERSION=local docker compose up
```

This will pick up the locally built image (with your locally built binary) and run tests against that image.

Reply:
No, this came up when I was trying to debug the TestACLs which was apparently failing. I added debug logs in the dgraph code but they didn't show up in the logs with this docker compose file because we don't mount the dgraph binary.

The usual workflow when I am testing things does not require me to run `make docker-image`. It would be odd to have that here and we may forget doing that while running or debugging tests.

This is what I would prefer because this is how we do things today. Though, I know things will change when I look at other client repos. So, if you are against the idea of mounting the binary, I can live with it for now. I will revisit this after going through other clients. your call.

Reply:
I see, that makes sense.  I agree that the development workflow should be consistent across the different repos.  I was looking at removing this also in the main dgraph repository but we can revisit this later.  I will change the workflow to inject the binary here like in the dgraph repo.